libraries/RC_Channel/RC_Channel.h

```diff
@@ -230,6 +230,7 @@ class RC_Channel {
         OPTFLOW_CAL =        158, // optical flow calibration
         FORCEFLYING =        159, // enable or disable land detection for GPS based manual modes preventing land detection and maintainting set_throttle_mix_max
         WEATHER_VANE_ENABLE = 160, // enable/disable weathervaning
+        TURB_START =         161, // initialize turbine start sequence
```

Comment:
I think we're missing the corresponding parameter description change to RCx_OPTION.